query-graphs/src/tree-rendering.ts

```diff
@@ -213,6 +213,7 @@ export function drawQueryTree(target: HTMLElement, treeData: TreeDescription) {
 
     // Initialize tooltip
     const tip = d3tip()
+        .attr("id", "tooltip")
```

Comment:
whenever possible we should try to avoid using an `id`.
In the JupyterLab plugin, multiple query-graphs will be rendered at the same time and then this Id won't be unique anymore. If we really need to use an id, let's append some timestamp/random number to it and thereby make it unique.

But do we really need an id here? Can't we just use a class instead?

Reply:
I'm removing it. I was using the ids mostly so I can quickly select the elements in browser `$("#id")`debugging and/or apply styles to them quickly.